README.md

```diff
@@ -82,7 +81,6 @@ export gluten_jvm_jar = /PATH/TO/GLUTEN/backends-velox/target/<gluten-jar>
 spark-shell 
   --master yarn --deploy-mode client \
   --conf spark.plugins=io.glutenproject.GlutenPlugin \
-  --conf spark.gluten.sql.columnar.backend.lib=velox or ch \
```

Comment:
Hi @baibaichen, this config looks useless. I am removing it. It is also useless to CH backend. Right?